packages.json

```
@@ -1,5 +1,4 @@
-[
-  {
+[{
```

```suggestion
[
  {
```
only suggesting this for myself :)